sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/FieldBuilder.java

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.search.documents;
+
+import com.azure.search.documents.models.ComplexSearchField;
+import com.azure.search.documents.models.SimpleDataType;
+import com.azure.search.documents.models.SimpleSearchField;
+
+/**
+ * Helper to build search field
+ */
+public class FieldBuilder {
```

Comment:
Where's the `build method that takes a type (class) and uses attributes to construct a bunch of fields, or even the whole index? That's what track 1 in .NET was doing.